server/src/packages/packages.service.ts

```diff
@@ -102,6 +127,7 @@ export class PackagesService {
       dcp_project,
       dcp_packageid,
       dcp_name,
+      _dcp_project_value,
```

Comment:
Isn't this already available in `dcp_project`? If so, is there a good request to add more code to get the project id?